lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -49,15 +49,17 @@ function runLighthouse(url, configPath, isDebug) {
   isDebug = isDebug || Boolean(process.env.LH_SMOKE_DEBUG);
 
   const command = 'node';
-  const outputPath = `smokehouse-${Math.round(Math.random() * 100000)}.report.json`;
-  const artifactsDirectory = './.tmp/smokehouse-artifacts';
+  const randInt = Math.round(Math.random() * 100000);
+  const outputPath = `smokehouse-${randInt}.report.json`;
+  const artifactsDirectory = `./.tmp/smokehouse-artifacts-${randInt}`;
   const args = [
     'lighthouse-cli/index.js',
     url,
     `--config-path=${configPath}`,
     `--output-path=${outputPath}`,
     '--output=json',
-    `-GA=${artifactsDirectory}`,
+    `-G=${artifactsDirectory}`,
+    `-A=${artifactsDirectory}`,
```

Comment:
any reason to split these up

Reply:
yup. 

> `node lighthouse-cli/index.js http://localhost:10200/seo/seo-tester.html?extra_header=link%3D%253Chttp%253A%252F%252Flocalhost%253A10200%252Fseo%252F%253E%253B%2Brel%253D%2522canonical%2522 --config-path=/Users/paulirish/code/pristine/lighthouse-pristine/lighthouse-cli/test/smokehouse/seo-config.js --output-path=smokehouse-83625.report.json --output=json -GA=./.tmp/smokehouse-artifacts-83625`

![image](https://user-images.githubusercontent.com/39191/57318494-52aa7000-70af-11e9-8dee-8eced320f10a.png)

Reply:
:this_is_fine: